includes/admin_import.php

```diff
@@ -248,243 +371,280 @@ function tsml_import_page()
+							<p><?php _e('<strong>Website</strong> and <strong>Website 2</strong> are optional.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Email</strong> is optional. This is a public email address.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Phone</strong> is optional. This is a public phone number.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Venmo</strong> is an optional string and should be a valid Venmo handle, (e.g. <code>@AAGroupName</code>). This is understood to be the address for 7th Tradition contributions to the meeting, and not any other entity.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Square</strong> is an optional string and should be a valid Square Cash App cashtag, (e.g. <code>$AAGroupName</code>). This is understood to be the address for 7th Tradition contributions to the meeting, and not any other entity.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>PayPal</strong> is an optional string and should be a valid PayPal username, (e.g. <code>AAGroupName</code>). This is understood to be the address for 7th Tradition contributions to the meeting, and not any other entity.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>URL</strong> is optional and should point to the meeting\'s listing on the area website.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Feedback URL</strong> is an optional string URL that can be used to provide feedback about the meeting. These could be local links (e.g. <code>https://mywebsite.org/feedback?meeting=meeting-slug-1</code>), remote links (e.g. <code>https://typeform.com/to/23904203?meeting=meeting-slug-1</code>), or email links (e.g. <code>mailto:webservant@domain.org?subject=meeting-slug-1</code>).', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Edit URL</strong> is an optional string URL that trusted servants can use to edit the specific meeting\'s listing.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Contact 1/2/3 Name/Email/Phone</strong> (nine fields in total) are all optional. By default, contact information is only visible inside the WordPress dashboard.', '12-step-meeting-list') ?></p>
+							<p><?php _e('<strong>Last Contact</strong> is an optional date.', '12-step-meeting-list') ?></p>
+							<?php if (!empty($tsml_programs[$tsml_program]['types'])) { ?>
+								<p><?php _e('<strong>Types</strong> should be a comma-separated list of the following options. This list is determined by which program is selected on the Settings tab.', '12-step-meeting-list') ?>
+								<ul class="types">
+									<?php foreach ($tsml_programs[$tsml_program]['types'] as $value) { ?>
+										<li><?php echo $value ?></li>
 									<?php } ?>
-								</p>
-								<input type="submit" class="button" value="<?php _e('Begin', '12-step-meeting-list') ?>">
-							</form>
-						</div>
+								</ul>
+								</li>
```

Comment:
Unnecessary closing tag. Can be removed.

Reply:
I'm a little confused with which closing tag you are referring to. Which one do you see as unnecessary?

Reply:
Line 621 </li>.

Reply:
Hilarious & Obvious. Why would a closing line item be outside the closing unordered list tag? It's gone now!
'